### PR TITLE
Develop upstream sync 220919

### PR DESCRIPTION
**Pull request by rahulbatra85**
Weekly upstream sync

### CODE REVIEW

**Comment by rahulbatra85** [comment]
retest Ubuntu-CPU please

File: tensorflow/compiler/xla/stream_executor/rocm/rocm_dnn.cc [hunk 1]
```diff
@@ -43,12 +43,8 @@ limitations under the License.
 #include "tensorflow/compiler/xla/stream_executor/scratch_allocator.h"
 #include "tensorflow/compiler/xla/stream_executor/stream.h"
 #include "tensorflow/compiler/xla/stream_executor/stream_executor_pimpl.h"
-<<<<<<< HEAD
```
Comment (rsanthanam-amd) [inline tensorflow/compiler/xla/stream_executor/rocm/rocm_dnn.cc]: We should probably keep this include.

It is needed for statements like: #if TF_ROCM_VERSION >= 50100

**Comment by rahulbatra85** [comment]
retest Ubuntu-CPU please

File: tensorflow/compiler/xla/stream_executor/rocm/rocm_dnn.cc (hunk 1 above)
Comment (rahulbatra85) [reply tensorflow/compiler/xla/stream_executor/rocm/rocm_dnn.cc]: @rsanthanam-amd  I have added it back

Comment (rsanthanam-amd) [reply tensorflow/compiler/xla/stream_executor/rocm/rocm_dnn.cc]: thanks

**Review by rsanthanam-amd — APPROVED** [review]

**Review by jayfurmanek — APPROVED** [review]
We'll need to upstream these fixes (along with other recent ones)
Good job! Looks good.

**Pull request merged**